### PR TITLE
Add web tracking instructions for custom projects.

### PR DESCRIPTION
**Pull request by mikeng13**
@jprovine @aliabbasrizvi @haleybash-optimizely @delikat @vraja2 @jophde @mauerbac check out the instructions for tracking on the web with custom projects

### CODE REVIEW

File: src/content/server/reference/tracking/2-web-tracking.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+template: multi-example
+title: Tracking on the web
+anchor: webtracking
+---
+
+If you'd like to track events on the web (via Javascript) instead of on the server, you can use the [Javascript SDK](https://github.com/optimizely/optimizely-testing-sdk-javascript):
```
Comment (aliabbasrizvi) [inline src/content/server/reference/tracking/2-web-tracking.md]: nit `JavaScript`

Comment (aliabbasrizvi) [inline src/content/server/reference/tracking/2-web-tracking.md]: I'd call it client side and server side. Thoughts @jprovine ?

Comment (aliabbasrizvi) [inline src/content/server/reference/tracking/2-web-tracking.md]: Customers will not be able to access it as it is a private repository. How about we ask them to email developers at optimizely dot com in the interim with the requirement and then we help them obtain it.

File: src/content/server/reference/tracking/2-web-tracking.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+1. Grab the `optimizely.min.js` file and include it in your Javascript bundle via webpack or your preferred bundling tool. You can also just add it as a script tag `<script type="text/javascript" src="../path_to/optimizely.min.js"`
```
Comment (aliabbasrizvi) [inline src/content/server/reference/tracking/2-web-tracking.md]: nit. `JavaScript`

**Comment by aliabbasrizvi** [comment]
This change although good is contingent upon a delivery mechanism to the end user. For now I'd recommend to email developers at optimizely dot com. Let me know your thoughts.

**Comment by mikeng13** [comment]
@aliabbasrizvi sounds good. Let's have them email developers at

**Pull request merged**